### PR TITLE
create questions via the api

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@Rizziepit ready for review

File: unicore/ask/service/questions_api.py [hunk 1]
```diff
@@ -20,9 +21,35 @@ class QuestionResource(object):
     def __init__(self, request):
         self.request = request
 
-    @view(renderer='json')
+    @view(renderer='json', schema=QuestionSchema)
```
Comment (Rizziepit) [inline unicore/ask/service/questions_api.py:24]: is `schema=QuestionSchema` a cornice thing? Does it return nice responses if the validation fails?

Comment (miltontony) [inline unicore/ask/service/questions_api.py:24]: Yes it is. [cornice](http://cornice.readthedocs.org/en/latest/validation.html#using-colander) wraps the validation errors in an errors dict. It also returns the deserialized content in `request.validated`

Comment (Rizziepit) [inline unicore/ask/service/questions_api.py:24]: Great. I'm stealing that

**Comment by Rizziepit** [comment]
+1

**Pull request merged**